package.json

```diff
@@ -123,7 +103,248 @@
     "webpack": "^5.72.1",
     "webpack-cli": "^4.9.2",
     "webpack-dev-server": "^4.9.0",
-    "webpack-stats-plugin": "^1.0.3"
+    "webpack-stats-plugin": "^1.0.3",
+    "victory-vendor": "^36.5.1",
+    "victory-voronoi": "^36.5.1",
+    "wireit": "^0.7.1"
+  },
+  "pnpm": {
+    "overrides": {
+      "@types/eslint": "8.4.3"
+    }
+  },
+  "scripts": {
+    "TODO_REFACTOR_start": "nps start",
+    "TODO_REFACTOR_start-ts": "nps start.ts",
+    "TODO_REFACTOR_storybook": "nps storybook.server",
+    "TODO_REFACTOR_build-storybook": "build-storybook",
+    "TODO_REFACTOR_chromatic": "chromatic",
+    "TODO_REFACTOR_chromatic-ci": "build-storybook && chromatic --ci --exit-zero-on-changes",
+    "check": "wireit",
+    "check:ci": "wireit",
+    "clean": "rimraf coverage && pnpm --recursive exec rimraf dist es lib",
```

Comment:
I think this one might be better done as `rimraf coverage "packages/*/{dist,es,lib}`.

Reply:
Yep, that sounds better :)